### PR TITLE
multi classing layout tweak

### PR DESCRIPTION
**Pull request by jasonmcalpin**
the other headings are part of multiclassing so I change them one level down.

### CODE REVIEW

File: source/license.rst [hunk 1]
```diff
@@ -155,13 +155,13 @@ Reserved.
...
-System Reference Document 5.0 Copyright 2016, Wizards of the Coast,
-Inc.; Authors Mike Mearls, Jeremy Crawford, Chris Perkins, Rodney
-Thompson, Peter Lee, James Wyatt, Robert J. Schwalb, Bruce R. Cordell,
-Chris Sims, and Steve Townshend, based on original material by E. Gary
-Gygax and Dave Arneson.
+    System Reference Document 5.0 Copyright 2016, Wizards of the Coast,
+    Inc.; Authors Mike Mearls, Jeremy Crawford, Chris Perkins, Rodney
+    Thompson, Peter Lee, James Wyatt, Robert J. Schwalb, Bruce R. Cordell,
+    Chris Sims, and Steve Townshend, based on original material by E. Gary
+    Gygax and Dave Arneson.
```
Comment (maxrothman) [inline source/license.rst]: Why these changes?

**Comment by maxrothman** [comment]
What's the gist of the change? Moving some sections up out of another section?

**Comment by jasonmcalpin** [comment]
Indent that paragraph of legal doc. Pretty minor.

Multiclassing. All headings that are part of it were changed 1 level down. So it was changed to this hierarchy:

Multi classing
	•	Prerequisites
	•	Experience Points
	•	Hit Points And Hit Dice
	•	Proficiency Bonus
	•	Proficiencies
	•	Multi Classing Proficiencies
	•	Class Features
	•	Channel Features
	•	Extra Attack
	•	Unarmed Defense
	•	Spell Casting

**Comment by jasonmcalpin** [comment]
I am looking this over again since it seems they mixed level related info among the info for multiclass. will post a revision later

**Comment by jasonmcalpin** [comment]
Not not moving anything.  Let the survey decide that.

**Comment by eepMoody** [comment]
I'd say it's good to go then. Definitely shouldn't all have been on the same level, so your changes would put it in line with how it's presented in the book.
